README.md

```diff
@@ -120,6 +120,15 @@ By default Phoenix imports `Ecto.Model` in the generated `ConnCase` and
 import Ecto.Model, except: [build: 2]
 ```
 
+If you want to keep the factories in the `test/factories` folder, change this line in mix.exs:
```

Comment:
What do you think of making this a bit more generic. Something like:

"If you want to keep the factories somewhere other than `test/support`, change this line in `mix.exs`"

Then down below:

`# Add the folder to the end of the list. In this case we're adding, `test/factories.`

Sorry to go back and forth on the suggestion for the comment!